WordPress/Classes/Utility/Analytics/WPAppAnalytics.h

```diff
@@ -38,7 +36,7 @@ extern NSString * const WPAppAnalyticsValueSiteTypeP2;
 /**
  *  @brief      Timestamp of the app's opening time.
  */
-@property (nonatomic, strong, readwrite) NSDate* applicationOpenedTime;
+@property (nonatomic, strong, readwrite) NSDate* _Nullable applicationOpenedTime;
```

Comment:
Nitpick: adding `nonnull` to the property attributes instead?

Reply:
Oh yeah forgot it about. Updated. It's much nicer.